css_01/css_01.css

```diff
@@ -0,0 +1,49 @@
+#bodyContent
+{
+	margin: auto;
+	width: 380px;
+}
+
+#items
+{
+	border: solid;
+	border-width: 1.5px;
+	border-color: gray;
+	background: #E8E8E8;
+	padding-top: 20px;
+	padding-left: 20px;
+	padding-bottom: 15px;
+	padding-right: 24px;
+}
+
+#title
+{
+	background: #66FF00;
+	text-align: center;
+	font-family: Sans-serif;
+	font-weight: bold;
+	color: white;
+	padding: 8px;
+}
+
+#email, #firstName, #lastName, #Password, #Confirm_Password
+{
+	width: 100%;
+	height: 25px;
+	font-size: 12px;
+	text-indent: 10px;
+	margin-bottom: 10px;
+}
+
+#item
```

Comment:
Here you need class. Please do not use the same id for different elements. https://css-tricks.com/almanac/selectors/i/id/ 

> ID attribute values should be unique. HTML with two or more identical ids does not validate, and will produce unpredictable results. If there are two of the same, CSS will still match and style both. JavaScript however, when querying for an ID, will find the first and stop.